gmso/tests/test_mcf.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+import mbuild
+import numpy as np
+import unyt as u
+
+import gmso
+from gmso.formats.mcf import write_mcf
+from gmso.tests.base_test import BaseTest
+from gmso.tests.utils import get_path
+from gmso.utils.io import get_fn
+from gmso.exceptions import EngineIncompatibilityError
+from gmso.external.convert_mbuild import from_mbuild
+
+
+class TestMCF(BaseTest):
+    def test_write_lj_mcf(self):
+        top = from_mbuild(mbuild.Compound(name="Ar"))
+
+        ff = gmso.ForceField(get_fn("ar.xml"))
+
+        for site in top.sites:
+            site.atom_type = ff.atom_types["Ar"]
+
+        top.update_topology()
+
+        write_mcf(top, "ar.mcf")
+
+        mcf_data = []
+        with open("ar.mcf") as f:
+            for line in f:
+                mcf_data.append(line.strip().split())
+
+        for idx, line in enumerate(mcf_data):
```

Comment:
I think it would also be good to add two simpler tests that only check that mie and LJ systems are written out correctly (So the test would end with `write_mcf()`.  

Are there plans to add a MCF reader as well?  I'm not super familiar with this file format so I don't know if there is value in that.  But I figure the tests that check that the information was written out correctly will be refactored if a reader is added as well.

Reply:
Thanks for the comments -- I don't personally see a major need for an MCF reader. I'm open to adding it if that is a desired feature. In that case, yes, I would refactor the tests.